src/app/modules/ix-feedback/feedback-dialog/feedback-dialog.component.html

```diff
@@ -43,7 +43,7 @@ <h1 matDialogTitle>{{ 'How would you rate this page?' | translate }}</h1>
       type="submit"
       color="primary"
       ixTest="submit"
-      [disabled]="form.invalid"
+      [disabled]="form.invalid || isLoading"
```

Comment:
It's missing a loading progress bar to indicate that something is loading.

Reply:
yeah, I just added disabled for now since it's out of the scope for this ticket, but I'll fix it!

Reply:
> It's missing a loading progress bar to indicate that something is loading.

agree, I just added at least disabled handling, since it was missing and is out of this scope, but I forgot to add loading visualisation.